docs/guides/05-legends-and-widgets.md

```diff
@@ -0,0 +1,240 @@
+
+The second parameter is optional, defaults to `1` and it is a way to weight each feature differently. For this case, we want to see the distribution of damage for each accident, without weighting each accident, but we could weight for example by the train size.
+
+The third parameter is the number of buckets of the histogram, by default it has a high value (1000), we'll simplify this a little bit here.
+
+The last step is to connect CARTO VL data with the Airship *as-histogram-widget* component. We do this with:
+```js
+layer.on('updated', drawHistogram);
+
+function drawHistogram() {
+    var histogramWidget = document.querySelector('as-histogram-widget');
+    const histogram = layer.viz.variables.histogram.value;
+    histogramWidget.data = histogram.map(entry => {
+        return {
+            start: entry.x[0] / 1e6,
+            end: entry.x[1] / 1e6,
+            value: Math.log(entry.y + 1)
+        }
+    });
+}
+```
+
+If you are curious about the usage of `Math.log`, this is done because the dataset has an exponential-like distribution that makes the histogram unusable without using a logarithmic scale.
+
+Let's see the full example!
+
+<div class="example-map">
+    <iframe
+        id="accidents-numeric-histogram"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-numeric-histogram.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+<a href="/developers/carto-vl/examples#example-railroad-accidents---numeric-histogram">View my source code!</a>
+
+##### Categorical histograms: how many votes did each party receive?
+
+To make a category histogram or widget the steps are similar.
+
+The Airship component is now *<as-category-widget>*:
+```html
+    <aside class="toolbox">
+        <div class="box">
+            <header>
+                <h1>Accident type</h1>
+            </header>
+            <as-category-widget />
+            <footer class="js-footer"></footer>
+        </div>
+    </aside>
+```
+
+The CARTO VL visualization can be simplified:
+```js
+const viz = new carto.Viz(`
+    @histogram: viewportHistogram($total_damage, 1, 6)
+`);
+```
+
+And the connection between Airship and CARTO VL needs a small adjustment:
+```js
+layer.on('updated', drawHistogram);
+
+function drawHistogram() {
+    var histogramWidget = document.querySelector('as-category-widget');
+    const histogram = layer.viz.variables.histogram.value;
+    histogramWidget.categories = histogram.map(entry => {
+        return {
+            name: entry.x,
+            value: entry.y
+        }
+    });
+}
+```
+
+<div class="example-map">
+    <iframe
+        id="accidents-category-histogram"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-category-histogram.html"
```

Comment:
We should deactivate the click on the widget, suggesting a filtering that doesn't happen

Reply:
:+1: